src/main/kotlin/no/nav/syfo/aktivitetskrav/api/AktivitetskravApi.kt

```diff
@@ -52,6 +52,15 @@ fun Route.registerAktivitetskravApi(
 
             call.respond(responseDTOList)
         }
+        post {
+            val personIdent = call.personIdent()
+            // val requestDTO: NewAktivitetskravDTO? = call.receiveNullable<NewAktivitetskravDTO>()
```

Comment:
Skal dette være kommentert ut?

Reply:
Nei, var WIP 😁 Pleier å sette PR i draft mode til den er klar for review, men det er ikke alltid like lett å se 😊